src/main/studio/kdb/Config.java

```diff
@@ -306,6 +306,33 @@ private void init(String filename, Properties properties) {
             }
         }
 
+        if (!Files.exists(file) && System.getProperty("os.name").startsWith("Windows")) {
+            System.out.println("Config not found in userprofile. Trying legacy path.");
```

Comment:
why not `log.info` ?

Reply:
Does that print to stdout _without any action from the user_ (e.g. providing log configuration)?

Reply:
System.out and System.err are trapped and also sent to Log4J (this is the reason of log4j-iostreams dependency). However this looses information about context (class name). So it looks more naturally to use logging.

Reply:
It turns out that log4j2.xml is not in the classpath. And this is because of the build.gradle:
```
            srcDirs 'src/main','images'
            include 'log4j2.xml'
```
It loads all resources from "include" (this one and all below) under src/main and images folder.

So one of the possible solution is to move log4j2.xml into src/main folder.

It is assumed that you should have studio.log and query.log files under ~/.studioforkdb/log/

Reply:
Thanks, moving the file fixes both issues.